### PR TITLE
Day 3: project machine learning

### PR DESCRIPTION
**Pull request by DaanVanHauwermeiren**

### CODE REVIEW

File: projects/machine_learning.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Machine learning: data-driven predictive modelling
+
+**tags:** *machine learning*
```
Comment (Beramos) [inline projects/machine_learning.md:3]: maybe add dataframes?

File: projects/machine_learning.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+```julia
+X = ...
+y = ...
+using MLJ
+models(matching(X, y))
+```
+
+4. If you are keen to try out neural networks: we recommend using the [Flux](https://fluxml.ai/) ecosystem of packages. 
+5. Consider using pipelines and target transformers for reproducible results!
```
Comment (Beramos) [inline projects/machine_learning.md:70]: can you add a url that gives some examples/documentation to `pipelines` and `target transformers`.

**Review by Beramos — APPROVED** [review]
Nice minor changes no need to review

File: projects/machine_learning.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Machine learning: data-driven predictive modelling
+
+**tags:** *machine learning*
+
+![project image](https://images.unsplash.com/photo-1580711508260-a9ea2b5377bb?q=80&w=2624&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D)
+
+## 1. Abstract
+
+Machine learning and data science: where computers learn from data like students cramming for exams, but without the coffee jitters. Picture a digital detective, tirelessly connecting the dots in a sea of numbers to reveal hidden patterns and insights. It's like teaching a robot to predict the future, but instead of a crystal ball, it uses algorithms and statistics. In this realm, data is the king, algorithms are its loyal subjects, and the quest is to turn chaos into clarity.
```
Comment (MichielStock) [inline projects/machine_learning.md:9]: 'We are highly confident this text was
ai generated"

**Comment by MichielStock** [comment]
I read this and can be merged.

**Pull request merged**